### ISSUE TITLE
New logo for ngx-rocket

### ISSUE THREAD

**Issue by sinedied** [ngx-rocket/generator-ngx-rocket#71]
After the [project renaming](https://github.com/ngx-rocket/generator-ngx-rocket/issues/65), we are looking for a new and nice logo to illustrate the ngX-Rocket ecosystem!

Here is a (bad) draft I thought up quickly:
![ngx-rocket-logo-poc](https://cloud.githubusercontent.com/assets/593151/26105805/dc38bb5a-3a43-11e7-8133-1d6d9848fcf8.png)

I you have better ideas and design skills, feel free to propose 🎨 !

**Comment by gaetanmaisse** [comment]
Here is my proposal based on @sinedied draft, any feedback is welcome!

![ngx-rocket](https://cloud.githubusercontent.com/assets/4112568/26123752/4bd85276-3a7c-11e7-9820-4d7edeb620fb.jpg)

**Comment by sinedied** [comment]
I :heart: it! here's my thoughts:

- I find it strange that `ngx` is not left (or right) aligned with the `rocket` text
- The `rocket` font is a bit too thin in my opinion
- I don't know if you've seen it, but in my draft I used the `A` from the Angular logo as the rocket flame :wink: I liked the idea, but maybe it's not suited to make it fit in the shield

We should make this a contest with something to win 👯 🍾

**Comment by sinedied** [comment]
I gave it another go based on @gaetanmaisse's design:
- I used roboto font (looks nicer than Angular's Sansara), but after looking at it again I still like better the thinner version of yours ^^
- I solved the left alignment problem with a `$` to remind of the CLI
- I changed the rocket a bit

**Comment by jycouet** [comment]
If I can add my comments, I would say that:
- I prefer the red logo :p
- The vertical rocket looks really weird

**Comment by sinedied** [comment]
Thanks for your feedback, everyone's opinion is welcome here!

**Comment by sinedied** [comment]
Here's a few ideas that could be used (I love the concept of last one with the cloud!)
![rocket-ideas](https://cloud.githubusercontent.com/assets/593151/26274012/70dc5fee-3d3e-11e7-9b3f-41ab55e6b7e7.png)

**Comment by nicolas-guerin** [comment]
Here's my proposal : first steps into logo design, so constructive feedback is welcome !

![ngx-rocket-banniere-font-small](https://cloud.githubusercontent.com/assets/8166873/26346415/77b77c3a-3fa6-11e7-9a2f-e22c91a0d1d0.jpg)

I didn't see your proposals before starting to work on mine : this is why my choices are so different, and maybe less adapted.

I just had a look at the end, and reused the Roboto font suggested by @gaetanmaisse.
To give my opinions on your proposals until now, I prefer the red version in the top bottom of @sinedied's first post : I prefer the vertical rocket (because placing it on the Angular shield, which is vertical and not diagonal), and I like the ionic-like electron 👍 , as it gives a global balance to the logo

**Comment by sinedied** [comment]
Nice try @nicolas-guerin ! It's even better if don't look at my proposals, it's not really a target on the opposite more original designs are welcome ❤️ 

There is some nice ideas in your proposal, I like the rocket trail and the blink at the top of it, but in my opinion by using a "full" background it make it more suited for landing page banner than a logo.

**Comment by sinedied** [comment]
I'm still not satisfied with any of the latest designs, but while waiting for an awesome designer to take the best of current ideas and make it a logo that everyone agrees on, I'll try the rocket/electron one just to get rid of that ugly package logo 😉 

I'll go with the blue to differentiate a bit from Angular and to remind the "enterprise" aspect, using the Angular red as base with a 140° hue roration:
![ngx-rocket-material-bi-650](https://user-images.githubusercontent.com/593151/27806815-1cf1dd74-603d-11e7-98aa-e6e100cbdac3.png)
![logo](https://user-images.githubusercontent.com/593151/27806819-20a159ae-603d-11e7-8544-7cc3ac1ecea0.png)

**Comment by gaetanmaisse** [comment]
As ngx-rocket is now a Thales official open source asset I rework a bit my first proposal to:
 - Add @sinedied idea of ionic like electron 
 - Update colors to use Thales one's

I find this dark blue version is less aggressive than the current one and I think it can be seen as a simple reminder of Thales identity (a very good point for internal adoption ;))

![fichier 1](https://user-images.githubusercontent.com/4112568/33126776-11fc22b8-cf86-11e7-899d-694be5fee548.png)

**Comment by bursauxa** [comment]
I believe the latest version from @sinedied is good enough. It is consistent with Angular designs while being different enough. If we all agree on it then we should close the issue.

**Comment by sinedied** [comment]
@bursauxa Yes this issue has lived long enough (if everyone is okay with that!). If needed, we'll open a new one in the future to change the logo 😉
